change/@fluentui-react-table-a86be917-7fe4-41a7-a5aa-19d22c2aa1f8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 76 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28097&commitId=d5373be5f5a43150c1a13a19536acbd2f5e035b6&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1868 screenshots unchanged, 0 screenshots with different dimensions and 76 screenshots with visible difference.


  <details><summary><b>unknown</b> 76 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|692| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|692| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|826| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|826| Changed|
|Table layout table - cell actions.always visible - RTL.chromium.png|746| Changed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|746| Changed|
|Table layout table - cell actions.always visible.chromium.png|710| Changed|
|Table layout table - cell actions.always visible.hover row.chromium.png|710| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|692| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|692| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|826| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|826| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|746| Changed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|746| Changed|
|Table layout table - cell actions.default.chromium.png|710| Changed|
|Table layout table - cell actions.default.hover row.chromium.png|710| Changed|
|Table layout table - cell actions.in header cell.chromium.png|2178| Changed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|2156| Changed|
|Table layout table - headers.sortable - Dark Mode.chromium.png|755| Changed|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|750| Changed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|753| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|902| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|845| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|845| Changed|
|Table layout table - headers.sortable.chromium.png|775| Changed|
|Table layout table - headers.sortable.hover header.chromium.png|772| Changed|
|Table layout table - headers.sortable.press header.chromium.png|765| Changed|
|Table layout table - truncate.default (disabled).chromium.png|946| Changed|
|Table layout table - truncate.false.chromium.png|946| Changed|
|Table layout table - truncate.true.chromium.png|946| Changed|
|Table layout table.keyboard column resizing style.chromium.png|710| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|692| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|826| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|746| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|710| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|692| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|826| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|746| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|710| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|692| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|826| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|746| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|710| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|692| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|826| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|746| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|710| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|692| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|826| Changed|
|Table layout table.multiselect - RTL.chromium.png|746| Changed|
|Table layout table.multiselect.chromium.png|710| Changed|
|Table layout table.primary cell.chromium.png|710| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|692| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|826| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|746| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|692| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|826| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|746| Changed|
|Table layout table.single select (checked) - brand.chromium.png|710| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|692| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|826| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|746| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|710| Changed|
|Table layout table.single select (checked).chromium.png|710| Changed|
|Table layout table.single select - Dark Mode.chromium.png|692| Changed|
|Table layout table.single select - High Contrast.chromium.png|826| Changed|
|Table layout table.single select - RTL.chromium.png|746| Changed|
|Table layout table.single select.chromium.png|710| Changed|
|Table layout table.size - extra small.chromium.png|512| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|692| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|826| Changed|
|Table layout table.size - medium - RTL.chromium.png|746| Changed|
|Table layout table.size - medium.chromium.png|710| Changed|
|Table layout table.size - small.chromium.png|710| Changed|
|Table table - subtle selection.rest.chromium.png|710| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|710| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>